Bottom: b7d7ea308c9bc9debf3ef071c790fce4f1c16d23
Top:    50b72a35999a2aba648054d4d43c4079502963a4
Author: Jon Smirl <jonsmirl@smirl2.(none)>
Date:   2013-06-22 14:57:06 -0400

Refresh of asiarf

---

diff --git a/target/linux/ramips/dts/AWM002-8M.dts b/target/linux/ramips/dts/AWM002-8M.dts
index adadfa1..6d82642 100644
--- a/target/linux/ramips/dts/AWM002-8M.dts
+++ b/target/linux/ramips/dts/AWM002-8M.dts
@@ -75,25 +75,6 @@
 		status = "okay";
 	};
 
-	i2c-gpio {
-		compatible = "i2c-gpio";
-
-		gpios = <&gpio0 1 0 &gpio0 2 0>;
-		i2c-gpio,delay-us = <10>;
-
-		pcf0: iexp@38 {
-			#gpio-cells = <2>;
-			compatible = "inxp,pcf8574a";
-			reg = <0x38>;
-			gpio-controller;
-		};
-
-		hwmon@4b {
-			compatible = "national,lm92";
-			reg = <0x4b>;
-		};
-	};
-
 	gpio-leds {
 		compatible = "gpio-leds";
 		ld1 {
